### PR TITLE
Add bosnian translations

### PR DESCRIPTION
**Pull request by jekkos**
This changeset adds a translation for bosnian.

### CODE REVIEW

**Review by cikov — APPROVED** [review]

**Review by cikov — APPROVED** [review]
I give permission to publish the Bosnian language

**Comment by jekkos** [comment]
@cikov we will need to wait for @narfbg to merge this one.. This is all we can do.

**Comment by cikov** [comment]
ok thanks a lot

**Comment by narfbg** [comment]
Sorry, but I've never had interest in the translations project. Never merged a single PR or anything, this very comment is I think my first action in this repo. This was entirely Jim's domain and as he is unfortunately no longer with us, I guess it just stalled.

Just now I invited @gxgpet as a maintainer here as well, but whether he wants to participate or not is entirely up to him.

**Comment by gxgpet** [comment]
Sure, can give a hand here as well! :D

File: language/bosnian/calendar_lang.php
Comment (gxgpet) [inline language/bosnian/calendar_lang.php:9]: As these are new files, let's ensure that the year in the copyright notice is the proper one.

File: language/bosnian/date_lang.php
Comment (gxgpet) [inline language/bosnian/date_lang.php]: This section doesn't seem to be translated. It's true that I see one another example as well, in another language, where this is not translated...

**Review by gxgpet — CHANGES_REQUESTED** [review]

File: language/bosnian/calendar_lang.php
Comment (jekkos) [reply language/bosnian/calendar_lang.php:9]: @cikov I guess you'll want to put your credentials here. Is your nickname sufficient?

Comment (cikov) [reply language/bosnian/calendar_lang.php:9]: Copyright (c) 2014 - 2021, British Columbia Institute of Technology

Comment (cikov) [reply language/bosnian/calendar_lang.php:9]: @jekkos nickname is sufficient

File: language/bosnian/date_lang.php
Comment (cikov) [reply language/bosnian/date_lang.php:55]: I made a translation
[date_lang.zip](https://github.com/bcit-ci/codeigniter3-translations/files/6046875/date_lang.zip)

**Review by cikov — APPROVED** [review]

**Review by cikov — APPROVED** [review]

File: language/bosnian/date_lang.php
Comment (cikov) [reply language/bosnian/date_lang.php:55]: [date_lang.zip](https://github.com/bcit-ci/codeigniter3-translations/files/6046743/date_lang.zip)
 made a translation

Comment (cikov) [reply language/bosnian/date_lang.php]: I made a translation
[date_lang.zip](https://github.com/bcit-ci/codeigniter3-translations/files/6046967/date_lang.zip)

Comment (cikov) [reply language/bosnian/date_lang.php]: @jekkos please do it. thanks!

**Review by cikov — APPROVED** [review]

File: language/bosnian/date_lang.php
Comment (jekkos) [reply language/bosnian/date_lang.php]: @gxgpet Translations were added

File: language/bosnian/calendar_lang.php
Comment (jekkos) [reply language/bosnian/calendar_lang.php:9]: @gxgpet Copyrights added.

**Comment by cikov** [comment]
> @jekkos commented on this pull request.
> 
> 
> 
>> +$lang['date_year'] = 'Godina';
>
+$lang['date_years'] = 'Godine';
> +$lang['date_month'] =
'Mjesec';
> +$lang['date_months'] = 'Mjeseci';
>
+$lang['date_week'] = 'Nedjelja';
> +$lang['date_weeks'] =
'Nedjelje';
> +$lang['date_day'] = 'Dan';
>
+$lang['date_days'] = 'Dani';
> +$lang['date_hour'] = 'Sat';
> +$lang['date_hours'] = 'Sati';
> +$lang['date_minute'] =
'Minut';
> +$lang['date_minutes'] = 'Minute';
>
+$lang['date_second'] = 'Sekunda';
> +$lang['date_seconds'] =
'Sekunde';
> +
> +$lang['UM12']	= '(UTC -12:00)
Baker/Howland Island';
> 
> @gxgpet Translations were
added
> 
> --
> You are receiving this because you
were mentioned.
> Reply to this email directly or view it on
GitHub:
>
https://github.com/bcit-ci/codeigniter3-translations/pull/431#discussion_r585976519


I sent a new (complete) translation for date_lang.php to
https://github.com/bcit-ci/codeigniter3-translations/pull/431/files so I
asked you to update it.
thanks a lot.

**Comment by jekkos** [comment]
@gxgpet I think I addressed all your comments. Let me know if anything else. Thanks.

**Comment by cikov** [comment]
Merged #431 into develop.  â
You are receiving this because you were mentioned.
Reply to this
email directly, view
it on GitHub, or unsubscribe.  

ok, thanks

**Comment by cikov** [comment]
Hi, @gxgpet please should be modified in /application/helpers/locale_helper.php
'bs:bosnian' => 'Bosnian', in 'bs-BA:bosnian' => 'Bosnian',

I have a new translation for:  /bower_components/bootstrap-table/dist/locale-all.js and locale-all.min.js
"bs-BA"
bootstrap-table-locale-all.min.js.zip
bootstrap-table-locale-all.zip
2)
for /bower_components/bootstrap-table/dist/locale/bs-BA.js and bs-BA.min.js
bootstrap-table-bs-BA.zip
bootstrap-table-bs-BA.min.zip
3)
for /bower_components/bootstrap-table/src/locale/bootstrap-table-bs-BA.js
bootstrap-table-bs-BA.zip

thanks

**Comment by cikov** [comment]
I sent this as a comment to https://github.com/bcit-ci/codeigniter3-translations/commit/b563d916cf3321b2f7b305f5238e41bcd2c692f2
5 days ago

**Comment by jekkos** [comment]
@cikov should be fixed now in latest opensourcepos#master

**Comment by gxgpet** [comment]
Sorry @cikov, but I can't work too much by using external ZIP files/resources or through comments diffs...
If you would like to improve an existing translation, kindly please open a new PR with the improvements and I will review it.

**Pull request merged**
